src/Composer/FixWpStubs.php

```diff
@@ -25,6 +25,11 @@ public static function php73Polyfill(Event $event): int
         $io->write('Removing duplicate is_countable() ...');
 
         $vendorDir = $event->getComposer()->getConfig()->get('vendor-dir');
+        if (!is_string($vendorDir) || strlen($vendorDir) <= 0) {
```

Comment:
Oh! 😱 negative-length strings :)

```suggestion
        if (!is_string($vendorDir) || strlen($vendorDir) !== '') {
```

Reply:
Those are the best strings. If you haven't worked with them, you haven't worked with strings!

Reply:
I have a special relations with strings.
https://gist.github.com/szepeviktor/6003fb6e2a35c0e5f9eb29c96623e4b7

Reply:
I'm a bit scared now.